.github/workflows/create_overlay.yaml

```diff
@@ -44,40 +44,36 @@ jobs:
           fetch-depth: 0
           clean: false
 
-      - name: Install dependencies
-        run: |
-          sudo snap install charmcraft --classic
-
       - name: Update bundle.yaml
         run: |
           echo "  legend-studio:"
           echo "    channel: ${{ steps.release-short.outputs.yyyy_mm }}/edge"
           echo "    resources:"
-          echo "      studio-image: ${{ github.event.inputs.studio_image }}" 
+          echo "      studio-image: ${{ github.event.inputs.studio-image }}" 
           
           # Create an overlay yaml file with the desired release version for Legend Engine,
           # SDLC, and Studio, and apply it over the bundle.yaml file.
-          overlay_file="overlay-$${{ github.event.inputs.release }}.yaml"
+          overlay_file="overlay-${{ github.event.inputs.release }}.yaml"
           echo "applications:" > $overlay_file
           echo "  legend-engine:" >> $overlay_file
           echo "    channel: ${{ steps.release-short.outputs.yyyy_mm }}/edge" >> $overlay_file
           echo "    resources:" >> $overlay_file
-          echo "      engine-image: ${{ github.event.inputs.engine_image }}" >> $overlay_file
+          echo "      engine-image: ${{ github.event.inputs.engine-image }}" >> $overlay_file
           echo "  legend-sdlc:" >> $overlay_file
           echo "    channel: ${{ steps.release-short.outputs.yyyy_mm }}/edge" >> $overlay_file
           echo "    resources:" >> $overlay_file
-          echo "      sdlc-image: ${{ github.event.inputs.sdlc_image }}" >> $overlay_file
+          echo "      sdlc-image: ${{ github.event.inputs.sdlc-image }}" >> $overlay_file
           echo "  legend-studio:" >> $overlay_file
           echo "    channel: ${{ steps.release-short.outputs.yyyy_mm }}/edge" >> $overlay_file
           echo "    resources:" >> $overlay_file
-          echo "      studio-image: ${{ github.event.inputs.studio_image }}" >> $overlay_file
+          echo "      studio-image: ${{ github.event.inputs.studio-image }}" >> $overlay_file
 
```

Comment:
We still need to apply the overlay, see: https://github.com/finos/legend-juju-bundle/blob/main/.github/workflows/create_release.yaml#L45-L48

Reply:
Actually, this action adds the overlay file, which can be later be used by people wanting to use specifically that release (we only have yyyy-mm tracks on charmhub, meaning that the second release in the month will override the first. The idea for the ``overlay.yaml`` file was to use that first release, if needed).